app.js

```diff
@@ -750,9 +750,14 @@ d.run(() => {
 		},
 		(err, scope) => {
 			// Receives a 'cleanup' signal and cleans all modules
-			process.once('cleanup', error => {
+			process.once('cleanup', (error, code) => {
 				if (error) {
 					logger.fatal(error.toString());
+					if (code === undefined) {
+						code = 1;
+					}
+				} else if (code === undefined || code === null) {
```

Comment:
Do we exit from the application intentionally only after finished `snapshot` process? We should add a comment here.

Reply:
OK, I added a comment.